### ISSUE TITLE
Identify recombinants which we are reasonably certain of, which have not previously been detected

### ISSUE THREAD

**Issue by hyanwong** [jeromekelleher/sc2ts-paper#213]
With the new system, we are producing fewer recombinants, each of which is hopefully more trustworthy. We should be able to compare the HMM likelihood with the equivalent value when no recombination is allowed (mismatch -> infinity). We can express this as an odds ratio, and pick the recombination nodes with high odds ratios, but which have no Pango-X designation, as ones of which we are reasonably certain but which were not previously identified.

**Comment by jeromekelleher** [comment]
Good idea. We don't actually get a likelihood out of the HMM though, just a path. We can compute the likelihood/probability posthoc though, as mu^{num_mismatches} etc.

**Comment by hyanwong** [comment]
Great: any way we can get a LR would be fine. I presume that the main way we could get misled is by time travellers. So it may be worth plotting the LR on the X axis against some measure correlated with our suspicion of time travel on the Y. I'm not sure what that measure would be, however.
